CHANGELOG.md

```diff
@@ -8,7 +8,9 @@ Prefix your items with `(Template)` if the change is about the template and not
 ## 3.10.X
 - Added Dependency Injection validation in the development environment.
 - Cleaned up the persistence configuration files (removed unused parameters and updated documentation).
+- Updated Contributing documentation.
```

Comment:
I don't think you updated the Contributing documentation.

Reply:
Tommy-Lee did not place it at the right place on his PR, I updated it.